index.html

```diff
@@ -122,88 +123,8 @@ <h5 class="card-title text-danger fw-bold">CC Ignite</h5>
 
     <section class="container-fluid bg-light m-5 p-5">
         <h3 class="text-center">Featured Speakers</h3>
```

Comment:
can you help me with

> Great design @DilsherB 🤝
> 
>     * [ ]  Kindly make sure your speakers' section is aligned into two columns as given in the design, and make sure all images appear for all speakers.
>       ![Speakers](https://user-images.githubusercontent.com/63917725/211008254-48457c30-5884-4748-947e-6a6cb6df1b76.PNG)
> 
>     * [ ]  Kindly remove this highlighted image from your desktop about page. This will make your design a replica of the design given.
>       ![about-issue](https://user-images.githubusercontent.com/63917725/211008874-e0348da1-ac82-4d91-b35a-2732fdca9c4a.PNG)
> 
>     * [ ]  You may also fix the responsiveness of your design. Kindly make sure your design can efficiently adopt to different screen sizes.

can you help me in this

Reply:
@1GORDON

Reply:
Hi @DilsherB, I suggest using the following  for their container:
`{`
`display:grid;`
`grid-templates-columns:1fr 1fr;`
`}`